hooks/post_gen_project.py

```diff
@@ -2,36 +2,44 @@
 # -*- coding: utf-8 -*-
 
 import logging
-logging.basicConfig(level=logging.DEBUG)
-logger = logging.getLogger('post_gen_project')
-
-import shutil
 import os
+import shutil
 
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger('post_gen_project')
 
-{% if cookiecutter.docs_tool == "mkdocs" %}
-
-logger.info('Moving files for mkdocs.')
-os.rename('mkdocs/mkdocs.yml', 'mkdocs.yml')
-shutil.move('mkdocs', 'docs')
-shutil.rmtree('sphinxdocs')
-
-{% elif cookiecutter.docs_tool == "sphinx" %}
-
-logger.info('Moving files for sphinx.')
-shutil.move('sphinxdocs', 'docs')
-shutil.rmtree('mkdocs')
-
-{% else %}
-
-logger.info('Removing all documentation files')
-shutil.rmtree('mkdocs')
-shutil.rmtree('sphinxdocs')
-
-{% endif %}
-
-logger.info('Removing all temporary license files')
-shutil.rmtree('licenses')
-
-logger.info('Removing jinja2 macros')
-shutil.rmtree('macros')
+DOC_SOURCES = 'doc_sources'
+ALL_TEMP_FOLDERS = [DOC_SOURCES, 'licenses', 'macros']
+DOC_TYPE_FILES_MAP = {
```

Comment:
I'd rather keep the files for each documentation tool separated, so that we don't have to deal with naming conflicts etc. 🤔

Reply:
They are separated exactly like before (in their own folders) - only now they all live in the `doc_sources` subfolder to make the tidy up process symmetric to licenses and macros (and fix the actual bug as a side effect of this :)).

Reply:
Ahh nice! Great 👍